v7/recent_posts_json/README.md

```diff
@@ -0,0 +1,22 @@
+This plug-in generates a /index.json file containing the same posts
+as /index.html. Intended to be used in JavaScripts. For example to
+promote the most recent posts at the bottom of older posts.
+
+By default, the JSON file will include:
+    {
+        <JS Date object>: {
```

Comment:
It should be made a bit more open and should be compatible with standard JSON parsers — in other words, use an ISO 8601 date or something more universal